### ISSUE TITLE
WebUI Completely Blank

### ISSUE THREAD

**Issue by Astranix593** [WangYihang/Platypus#226]
Built the program as instructed, and after fixing loads of errors the webUI is completely blank and white.

These errors in the console 

```Platypus.js:23 Uncaught TypeError: Cannot read properties of undefined (reading 'headers')
    at 336 (Platypus.js:23:16)
    at a ((index):1:1203)
    at r ((index):1:1072)
    at Array.t [as push] ((index):1:935)
    at main.593117ae.chunk.js:1:83```

**Comment by WangYihang** [comment]
Thanks for reporting, I guess that may due to the JavaScript dependency issue since I blindly merged several pull requests from dependabot.

**Comment by WangYihang** [comment]
The frontend is under refactor, cuz I was busy for a long time, could you please rollback your source code to the latest release?

**Comment by taomujian** [comment]
axios版本升级导致的，回滚到axios 0.21.2就可以了

**Comment by WangYihang** [comment]
Thank you so much for the contribution. @Astranix593 Please check it out.

Fixed in d01107a55f77a0958c1500481bbc09ade63325ef.
